src/v/raft/heartbeat_manager.cc

```diff
@@ -149,8 +163,17 @@ heartbeat_manager::send_heartbeats(std::vector<node_heartbeat> reqs) {
 }
 
 ss::future<> heartbeat_manager::do_dispatch_heartbeats() {
-    auto reqs = requests_for_range(_consensus_groups, _heartbeat_interval);
-    return send_heartbeats(std::move(reqs));
+    auto [reqs, resets] = requests_for_range(
+      _consensus_groups, _heartbeat_interval);
+
+    for (const auto& node_id : resets) {
+        if (co_await _client_protocol.ensure_disconnect(node_id)) {
```

Comment:
no need to check, this log is not useful as warn, maybe use a trace.

Reply:
I've downgraded this to info (we already have warn-level messages from subsequent reconnection attempts).  The thing I want to communicate with this log is that *we* tore down the connection, rather than the operating system.

Reply:
right but there is no need for bool, so you can make it unconditional log.

Reply:
actually nvm! ha i see what you did.